File: example8/fetch-stdout.py
#!/usr/bin/env python

import sys
import flux
from flux import kz

f = flux.Flux ()
kz.attach (f, "lwj.0.0.1.0.stdout", sys.stdout)
kz.attach (f, "lwj.0.0.1.1.stdout", sys.stdout)

f.reactor_run (f.get_reactor (), 0)

# vi: ts=4 sw=4 expandtab

File: example8/fetch-stdout.lua
#!/usr/bin/env lua
local flux = require 'flux'
local f = assert (flux.new())

local iow1, err = f:iowatcher {
                      key = "lwj.0.0.1.0.stdout",
                      handler = function (iow1, data)
                          if data ~= nil then
                            io.stdout:write (string.format ("%s", data))
                          end
                      end
                 }
local iow2, err = f:iowatcher {
                       key = "lwj.0.0.1.1.stdout",
                       handler = function (iow2, data)
                          if data ~= nil then
                            io.stdout:write (string.format ("%s", data))
                          end
                       end
                   }

f:reactor ()

-- vi: ts=4 sw=4 expandtab
